pilot/pkg/security/authn/policy_applier.go

```diff
@@ -32,7 +32,7 @@ type PolicyApplier interface {
 
 	// JwtFilter returns the JWT HTTP filter to enforce the underlying authentication policy.
 	// It may return nil, if no JWT validation is needed.
-	JwtFilter() *hcm.HttpFilter
+	JwtFilter(useExtendedJwt, clearRouteCache bool) *hcm.HttpFilter
```

Comment:
clearRouteCache looks weired

Reply:
Suggestions? That's what xDS field is called.

Reply:
actually i am not familiar with jwt clear cache, will take a look though

Reply:
It is to clear the cache used for making route decision, right? @kyessenov

Reply:
Yes, it's to re-calculate the route selection